core/Db/Schema/Mysql.php

```diff
@@ -75,6 +75,7 @@ public function getTablesCreateSql()
                             `group` VARCHAR(250) NOT NULL,
                             `type` VARCHAR(255) NOT NULL,
                             keep_url_fragment TINYINT NOT NULL DEFAULT 0,
+                            user_created VARCHAR(100) NULL,
```

Comment:
would it be `login_created` or `user_login_created` or `login_creator` or something? In the user table it is the `login` column that matches it. Might be more clear? not sure...

Reply:
Can change it to `creator_login` as in "the creator's login"? Was trying to be consistent w/ the `ts_created` column w/ `user_created`.

Reply:
`creator_login ` 👍 I was actually just thinking before the same. otherwise sounds like a user was created